cypress/integration/visitorViewsMessageWhenNoArticles.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('Visitor can view message on the landing page', () => {
+
+  it('if there are no articles in the API database', () => {
```

Comment:
you can have this in the other file, they are separated by the `it` block

Reply:
done